### ISSUE TITLE
[Bug]: Server Crashes with Ticking Entity when player joins server.

### ISSUE THREAD

**Issue by Xlaits** [TeamLapen/Vampirism#1326]
### Minecraft Version

1.20.1

### Forge Version

47.2.0

### Vampirism Version

1.10.3

### Modpack

https://legacy.curseforge.com/minecraft/modpacks/kodi-pack

### Singleplayer/Multiplayer

Singleplayer

### What happened?

Upon Joining, the server causes a Ticking Entity event, and immediately crashes.

### Relevant log output

```shell
[crash-2024-01-17_10.06.15-server.txt](https://github.com/TeamLapen/Vampirism/files/13964986/crash-2024-01-17_10.06.15-server.txt)
```

### Reproduce steps

1. Start Server
2. Join Server
3. Crash Server

### Other relevant mods

See Modpack linked.

**Comment by Xlaits** [comment]
[crash-2024-01-17_10.06.15-server.txt](https://github.com/TeamLapen/Vampirism/files/13965019/crash-2024-01-17_10.06.15-server.txt)

**Comment by Xlaits** [comment]
I have no idea why that did what it did, but I have uploaded the full crashlog here.

**Comment by lunofe** [comment]
This might have been fixed in Vampirism 1.10.5 - please update and try again.

**Comment by Cheaterpaul** [comment]
Duplicate of #1320
